www/%team/index.html.spt

```diff
@@ -18,7 +18,9 @@ is_team_owner = not user.ANON and team.owner == user.participant.username
 {% block head %}
     <link rel="payment" type="text/html" title="Give weekly gift on Gratipay"
           href="https://gratipay.com/{{ team.slug }}/" />
+    <meta name="ROBOTS" content="NOINDEX, NOFOLLOW" />
```

Comment:
We'd only index approved teams.

> What problem do you see with someone googling and coming on to a rejected/pending team page?

It can make the team look bad: https://github.com/gratipay/team-review/issues/54.